ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -133,22 +131,17 @@ export const PodsLogsViewer = (props: PodLogsProps) => {
         (viewTimestamps ? (lineNum === 0 || (logs[lineNum - 1].timeStamp !== log.timeStamp ? log.timeStampStr : '').padEnd(30)) + ' ' : '') +
         // show the log content, highlight the filter text
         log.content?.replace(highlight, (substring: string) => whiteOnYellow + substring + reset);
-
-    const cellRenderer = ({rowIndex, key, style}: GridCellProps) => {
-        return (
-            <pre key={key} style={style} className='noscroll'>
-                <Ansi>{renderLog(logs[rowIndex], rowIndex)}</Ansi>
-            </pre>
-        );
-    };
-
-    // calculate the width of the grid based on the longest log line
-    const maxWidth =
-        14 *
-        logs
-            .map(renderLog)
-            .map(v => v.length)
-            .reduce((a, b) => Math.max(a, b), 0);
+    const logsContent = (width: number, height: number, isWrapped: boolean) => (
+        <div style={{width, height, overflow: 'scroll'}}>
```

Comment:
where did `<Grid/>` go, I don't think this can work correctly without it

Reply:
Hi @alexec , thanks for the review. The <Grid> component has a fixed lineHeight which doesn't work well with the wrapping function. Have you seen the screenshot video I attached to the first comment? I have tested with long log content as well. Feel free to pull this to your local and test out with more cases.